wix-inbox-v2/wix-inbox-v2/guides/EDM-intro.md

```diff
@@ -20,11 +20,8 @@ title: Introduction
 > unless specified otherwise.
 
 
-With Wix Inbox v2, site contributors
-can communicate with the site's visitors, contacts, and members
-through Wix Chat, SMS, a Facebook business page, or other channels.
-Visitor activities on the site and 3rd-party services
-can also be displayed in Inbox conversations.
+With Wix Inbox v2, site collaborators can communicate with the site's visitors, contacts, and members
+through various channels such as Wix Chat, SMS, a Facebook business page, and more.
 
 The Inbox APIs expose functionality for working with a site's inbox.
 With the Inbox APIs, you can:
```

Comment:
I know you didn't write this, but I think this needs an extra blank line above for spacing.